projects/storefrontapp-cds-integration/src/environments/environment.ts

```diff
@@ -0,0 +1,9 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=prod` then `environment.prod.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: false,
+  occBaseUrl: 'https://dev-com-14.accdemo.b2c.ydev.hybris.com:9002/',
```

Comment:
Can you please update the cds todo list, and mention that this value needs to match with the same value from the main shell app's `environment.ts`?